### PR TITLE
Refactor/darkmode

### PR DESCRIPTION
**Pull request by tonycarpenter21**
What (if any) features are you implementing?
- Fix merge conflict and adjust style for dark mode

What (if anything) did you refactor?
- Dark mode and the interface file

Were there any issues that arose?
- No

Is there anything that you need from your teammate?
- No

Any other comments, questions, or concerns?
- No

### CODE REVIEW

File: src/interface.tsx [hunk 1]
```diff
@@ -4,6 +4,11 @@ export interface AppState {
   isDarkModeOn: boolean;
 }
 
+export interface NavProps {
+  darkMode: string | null;
+  toggleDarkMode: () => void;
+}
+
```
Comment (mr-ryan12) [inline src/interface.tsx:11]: Thank you for moving this into the interface file! Way to keep the structure of our files more modular!

File: src/sun.svg [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<path d="M12 17C14.7614 17 17 14.7614 17 12C17 9.23858 14.7614 7 12 7C9.23858 7 7 9.23858 7 12C7 14.7614 9.23858 17 12 17Z" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M12 1V3" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M12 21V23" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M4.21997 4.21997L5.63997 5.63997" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M18.36 18.36L19.78 19.78" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M1 12H3" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M21 12H23" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M4.21997 19.78L5.63997 18.36" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+<path d="M18.36 5.63997L19.78 4.21997" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+</svg>
```
Comment (mr-ryan12) [inline src/sun.svg:11]: Nice addition of the icon for dark-mode!

File: src/ArtDetails/ArtDetails.tsx [hunk 3]
```diff
@@ -4,14 +4,13 @@ import '../ArtView/ArtView.scss'
...
 import { motion } from "framer-motion"
-import { ArtDetailsProps } from '../interface'
 
 const ArtDetails: React.FC<ArtDetailsProps> = props => {
   const likedMessage = props.isLiked ? <p className="percent-message">You have great taste!</p>
                                      : <p className="percent-message">You have amazing taste!</p>
   return (
     <motion.section 
-      {"art-details-container" + props.darkMode}
+      className={"art-details-container" + props.darkMode}
```
Comment (StevieAmb) [inline src/ArtDetails/ArtDetails.tsx:13]: Awesome fix, Tony! Great job figuring this out!

File: src/Navigation/Navigation.scss [hunk 4]
```diff
@@ -51,6 +51,9 @@
   font-size: 2.9vh;
   background: none;
   cursor: pointer;
+  position: absolute;
+  top: 30px;
+  right: 20px;
```
Comment (mr-ryan12) [inline src/Navigation/Navigation.scss:56]: Awesome way to position the dark mode icon! Thank you for teaching me this!

File: src/Navigation/Navigation.scss [hunk 5]
```diff
@@ -68,5 +71,11 @@
   .angled-end-dark {
     display: none;
   }
+  .behind-logo {
+    max-width: 100%;
+  }
+  .dark-mode-button {
+    top: 40px;
+  }
```
Comment (StevieAmb) [inline src/Navigation/Navigation.scss:79]: Great work!

**Review by StevieAmb — APPROVED** [review]

**Pull request merged**